frontend/constants/constants.js

```diff
@@ -0,0 +1,15 @@
+const constants = {
```

Comment:
Not a big deal, but do we need this to be nested inside a `constants/` directory if we only have one file?